src/main/java/com/ucm/ms/accounts/controllers/UserAccountsController.java

```diff
@@ -53,17 +56,15 @@ public ModelAndView confirmUserAccount(ModelAndView modelAndView, @RequestParam(
         return modelAndView;
     }
     
-    /**
-     * GET /api/user_account - Return all user accounts
-     * @return All user accounts
-     */
-    @GetMapping
-    public ResponseEntity<Collection<UserAccount>> get() {
-        try {
-            Collection<UserAccount> userAccounts = userAccountSearch.getAll();
-            return new ResponseEntity<>(userAccounts, HttpStatus.valueOf(200));
-        } catch(Exception e) {
-            return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
-        }
-    }
+    
+	@GetMapping("/accounts")
```

Comment:
The URI here would be /api/user_accounts/accounts. It looks like this one gets all user accounts for a specific user. I'm not a fan of the URI path tbh but I can't think of any suggestions either.